Sources/MapboxSearch/PublicAPI/FavoriteRecord.swift

```diff
@@ -105,6 +109,7 @@ public struct FavoriteRecord: IndexableRecord, SearchResult, Codable, Equatable
         serverIndex: Int?,
         accuracy: SearchResultAccuracy?,
         categories: [String]?,
+        categoryIds: [String]?,
```

Comment:
This is a breaking change.

Reply:
Thank you.
Yes, we missed that. We need to add a default value for this parameter